x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/new_index/new_index.tsx

```diff
@@ -52,7 +52,7 @@ export const NewIndex: React.FC = () => {
       pageHeader={{
         description: i18n.translate('xpack.enterpriseSearch.content.newIndex.pageDescription', {
           defaultMessage:
-            'Create a search optimized Elasticsearch index by selecting an ingestion method',
+            'Create a search optimized Elasticsearch index by selecting an ingestion method. Start by selecting an ingestion method.',
```

Comment:
```suggestion
            'Create a search optimized Elasticsearch index to store your content. Start by selecting an ingestion method.',
```

Reply:
typo duplication

Reply:
@efegurkan if this runs too long, can revert to original:

Create a search optimized Elasticsearch index by selecting an ingestion method.

Reply:
![Screenshot 2023-05-11 at 11 25 49](https://github.com/elastic/kibana/assets/1410658/29d22bd9-928b-4526-94f2-2effdb81de47)
That looks perfect